breaking/entrypoint.sh

```diff
@@ -4,18 +4,21 @@ set -e
 readonly base="$1"
 readonly revision="$2"
 readonly fail_on_diff="$3"
-readonly include_checks="$4"
-readonly include_path_params="$5"
-readonly deprecation_days_beta="$6"
-readonly deprecation_days_stable="$7"
-readonly exclude_elements="$8"
+readonly fail_on="$4"
+readonly include_checks="$5"
+readonly include_path_params="$6"
+readonly deprecation_days_beta="$7"
+readonly deprecation_days_stable="$8"
+readonly exclude_elements="$9"
 
-echo "running oasdiff breaking... base: $base, revision: $revision, fail_on_diff: $fail_on_diff, include_checks: $include_checks, include_path_params: $include_path_params, deprecation_days_beta: $deprecation_days_beta, deprecation_days_stable: $deprecation_days_stable, exclude_elements: $exclude_elements"
+echo "running oasdiff breaking... base: $base, revision: $revision, fail_on_diff: $fail_on_diff, fail_on: $fail_on, include_checks: $include_checks, include_path_params: $include_path_params, deprecation_days_beta: $deprecation_days_beta, deprecation_days_stable: $deprecation_days_stable, exclude_elements: $exclude_elements"
 
 # Build flags to pass in command
 flags=""
 if [ "$fail_on_diff" = "true" ]; then
     flags="${flags} --fail-on WARN"
```

Comment:
should be removed

Reply:
@effoeffi For https://github.com/oasdiff/oasdiff-action/pull/43#discussion_r1516173477, the reason it was not removed was that it would be a breaking change itself if it was removed from the action. If someone relies on the `fail_on_diff` and it is removed, their workflows will break... of course `fail-on` is more flexible for any status. Will update the `pr` to remove `fail_on_diff` though.

Reply:
> @effoeffi For [#43 (comment)](https://github.com/oasdiff/oasdiff-action/pull/43#discussion_r1516173477), the reason it was not removed was that it would be a breaking change itself if it was removed from the action. If someone relies on the `fail_on_diff` and it is removed, their workflows will break... of course `fail-on` is more flexible for any status. Will update the `pr` to remove `fail_on_diff` though.

@jgill-shipwell that is why we are creating a release, and you should rely on it to avoid any breaking changes.

Reply:
@effoeffi Updated this `pr` to remove `fail_on_diff` and match the `fail-on` param of `oasdiff`. Updated the tests, etc.